### ISSUE TITLE
Screen Reader cannot read names of some elements in the 'Tips and tricks' screen

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7630]
The Tips and tricks screen has the following issues flagged by Accessibility Insights – see screen shot.

!image-20230729-000941.png|width=1555,height=824!

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/VPN-5292)
┆Reporter: Vinoo Cherian

**Comment by data-sync-user** [comment]
➤ Vinoo Cherian commented:

This screen has changed, so the issue is no longer valid.

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

Moving back to Cancelled
